README.md

```diff
@@ -1,25 +1,70 @@
-
-The ePayment SDK provides reactive clients for interacting with the ePayment API using the Java Flow API.<br>
-Currently there are no artifacts provided, so you would need to build them yourself using
->mvn install
-
-and then importing them into your project.
-## Usage
-
-### Importing
-#### Standard usage (Provided client)
-By importing the specific client(s) (e.g. **merchant-client**) you need and the **webflux-client** module - Everything should work out of the box. 
-
-#### Use your own http client
-If you don't want to use our HTTP client implementation using webflux, you are free to provide your own. <br>
-This SDK uses a Java Service Provider Interface. <br>
-You need to implement [HttpClient](java/base-client/src/main/java/no/bankaxept/epayment/client/base/http/HttpClient.java) and [HttpClientProvider](java/base-client/src/main/java/no/bankaxept/epayment/client/base/spi/HttpClientProvider.java)
-and then create a configuration file (like [this](java/webflux-client/src/main/resources/META-INF/services/no.bankaxept.epayment.client.base.spi.HttpClientProvider)). The [webflux-client module](java/webflux-client) can be used for reference
-
-### Starting clients
-It's pretty simple to create a client.<br>
-You need a few values to get started
-* **Base url** of the ePayment service
-* **Credentials for fetching tokens** .
-> var merchantClient = new MerchantClient(baseurl, username, password);
+# Development kit for BankAxept ePayment integration
+
+The BankAxept ePayment development kit provides OpenAPI specifications and Java clients for integration with the
+ePayment APIs:
+* Merchant API
+* Token Requestor API
+* Wallet API
+
+## Maven artifacts for Java clients
+
+There are three Maven artifacts available, one for each if the ePayment APIs:
+```xml
+<dependency>
+  <groupId>no.bankaxept.epayment</groupId>
+  <artifactId>merchant-client</artifactId>
+  <version>x.y.z</version>
+</dependency>
+
+<dependency>
+  <groupId>no.bankaxept.epayment</groupId>
+  <artifactId>token-requestor-client</artifactId>
+  <version>x.y.z</version>
+</dependency>
+
+<dependency>
+  <groupId>no.bankaxept.epayment</groupId>
+  <artifactId>wallet-client</artifactId>
+  <version>x.y.z</version>
+</dependency>
+```
+These artifact rely on an HTTP client loaded through Java's Service Provider Interface. There's one provided by thiskit
+development kit, based on Spring WebFlux:
+```xml
+<dependency>
+  <groupId>no.bankaxept.epayment</groupId>
+  <artifactId>webflux-client</artifactId>
+  <version>x.y.z</version>
+</dependency>
+```
+The artifacts are available on a GitHub Packages repository, so make sure to include the following in your pom.xml as
+well:
+```xml
+<repositories>
+  <repository>
+    <id>github</id>
+    <url>https://maven.pkg.github.com/BankAxept/bankaxept-epayment-development-kit</url>
+  </repository>
+</repositories>
+```
+_Note: GitHub Packages does not allow anonymous access for downloading Maven artifacts. A GitHub personal access
```

Comment:
Is this a different approach from what you mentioned on daily today, or am I just confused?

Reply:
Yes, GitHub App can't be used, as OAuth2 with client credentials is not supported by Maven.